pybb/views.py

```diff
@@ -443,6 +445,16 @@ def dispatch(self, request, *args, **kwargs):
                     return HttpResponse(self.quote)
         return super(AddPostView, self).dispatch(request, *args, **kwargs)
 
+    def post(self, request, *args, **kwargs):
+        response = super(AddPostView, self).post(request, *args, **kwargs)
+
+        if self.object and self.topic:
+            topic_updated.send(Post, post=self.object, request=self.request)
```

Comment:
Signal could be triggered into the `form_valid` method from `PostEditMixin` just after `self.object.save()`